.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+on:
```

Comment:
- [ ] Let's capture this in the RELEASE_NOTES.md as "Move CI from Travis to Github Actions and add CD"